quick_start/my_text_classifier_bert.jsonnet

```diff
@@ -0,0 +1,53 @@
+local bert_model = "bert-base-uncased";
+
+{
+    "dataset_reader" : {
+        "type": "classification-tsv",
+        "tokenizer": {
+            "type": "pretrained_transformer",
+            "model_name": bert_model,
+        },
+        "token_indexers": {
+            "bert": {
+                "type": "pretrained_transformer",
+                "model_name": bert_model,
+            }
+        },
+        "max_tokens": 512
+    },
+    "train_data_path": "data/movie_review/train.tsv",
+    "validation_data_path": "data/movie_review/dev.tsv",
+    "model": {
+        "type": "simple_classifier",
+        "embedder": {
+            "allow_unmatched_keys": true,
+            "embedder_to_indexer_map": {
+                "bert": ["bert", "mask"]
+            },
+            "token_embedders": {
+                "bert": {
+                    "type": "pretrained_transformer",
+                    "model_name": bert_model
+                }
+            }
+        },
+        "encoder": {
+            "type": "bert_pooler",
+            "pretrained_model": bert_model,
+            "requires_grad": false
+        }
+    },
+    "iterator": {
+        "type": "basic",
```

Comment:
Just a heads up, we're probably going to be removing the `BasicIterator`.  Probably we'll make it so you don't have to specify a type here, and we can safely remove this key.  It's fine with me to leave this here for now, this is just a note that this will probably have to change before the course is released.